modules/reference/pages/rpk/rpk-cluster/rpk-cluster-health.adoc

```diff
@@ -23,7 +23,7 @@ rpk cluster health [flags]
 |===
 |*Value* |*Type* |*Description*
 
-|-e, --exit-when-healthy |- |When used with watch, exits after cluster
+|-e, --exit-when-healthy |- |Exits after cluster
```

Comment:
```suggestion
|-e, --exit-when-healthy |- |Exits when the cluster
```

Reply:
this is different from the generated content: https://github.com/redpanda-data/docs/pull/285/files#diff-1596ce07c033aa828075b3650363061083af0d6171b5ce2b44375f2a56add51b

Reply:
Same as the other